src/applications/yellow-ribbon/actions/index.js

```diff
@@ -87,14 +87,20 @@ export const fetchResultsThunk = (options = {}) => async dispatch => {
     // Track the API request.
     if (trackSearch) {
       recordEvent({
-        event: 'edu-yellow-ribbon-search',
-        'edu-yellow-ribbon-name': name || undefined,
-        'edu-yellow-ribbon-state': state || undefined,
-        'edu-yellow-ribbon-city': city || undefined,
-        'edu-yellow-ribbon-contribution-amount':
-          contributionAmount || undefined,
-        'edu-yellow-ribbon-number-of-students': numberOfStudents || undefined,
-        'edu-yellow-ribbon-number-of-search-results': response?.totalResults,
+        event: 'view_search_results',
+        'search-page-path': '/education/yellow-ribbon-participating-schools/',
+        'search-query': name,
+        'search-typeahead-enabled': false,
+        'search-selection': 'Yellow Ribbon',
+        'search-filters-list': {
+          stateOrTerritory: state || undefined,
```

Comment:
I think we usually use `null` here for when it's missing for analytics, but worth confirming with @jonwehausen 🙂